src/core/thread/mesh_forwarder.cpp

```diff
@@ -2210,10 +2215,21 @@ void MeshForwarder::LogIp6Message(MessageAction aAction, const Message &aMessage
     {
     case kMessageReceive:
         actionText = "Received";
+        mIpCounters.mRxSuccess++;
         break;
 
     case kMessageTransmit:
-        actionText = (aError == OT_ERROR_NONE) ? "Sent" : "Failed to send";
+        if (aError == OT_ERROR_NONE)
+        {
+            actionText = "Sent";
+            mIpCounters.mTxSuccess++;
```

Comment:
I suggest we make counters available independent of the log level.
We can increase the values from where `LogIp6Message` is called.